.circleci/config.yml

```
@@ -42,6 +42,11 @@ jobs:
          command: |
            scripts/verify-link-anchors.sh
 
+      - run:
+         name: "Check unclosed tags"
+         command:
+          python3 scripts/check-tags.py $(git diff-tree --name-only --no-commit-id -r upstream/release-4.0..HEAD -- '*.md' ':(exclude).github/*' ':(exclude)v1.0/*' ':(exclude)v2.0/*' ':(exclude)v2.1-legacy/*')
```

```suggestion
          python3 scripts/check-tags.py $(git diff-tree --name-only --no-commit-id -r upstream/release-4.0..HEAD -- '*.md' ':(exclude).github/*')
```
git diff-tree --name-only --no-commit-id -r upstream/release-4.0..HEAD -- '*.md' 的准确意思是 **当前 branch** 与 upstream/release-4.0 branch 的 md 文件的不同吗？ @aylei

---

是的

---

好的，感谢